commcare_connect/templates/reports/uservisit_dashboard.html

```diff
@@ -0,0 +1,279 @@
+{% extends "base.html" %}
+{% load crispy_forms_tags %}
+{% block javascript %}
+{{ block.super }}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/echarts/5.4.3/echarts.min.js"></script>
```

Comment:
Shouldn't this be added as a local dependency?

I'm also interested to know how you chose this library vs the others out there.

Reply:
One of the main requirements for this dashboard is to be able to customize the appearance to a high degree (focused on external demos), which this library is capable of. It has [theme builder](https://echarts.apache.org/en/theme-builder.html) using which its possible to change a lot of styling. It's also used by Superset and so there's lot of parity in terms of being able to create whatever charts that are possible to build on superset (which the project team uses to build various charts/dashboards currently for internal use)

> Shouldn't this be added as a local dependency?

Yes, I will modify.